src/react/ReactPropsSet.js

```diff
@@ -34,6 +34,11 @@ export class ReactPropsSet extends ReactSet {
       currentMap = result.map;
     }
 
+    if (props.temporalAlias !== undefined) {
+      currentMap = this._getKey("temporalAlias", currentMap, visitedValues);
+      result = this._getValue(props.temporalAlias, currentMap, visitedValues);
+    }
```

Comment:
What case does this handle? Do tests fail without it?

Reply:
What if there's a prop called `temporalAlias`? Not likely but seems like an easy fix.